### DISCUSSION TITLE
Is there any way to pass capture info from quickadd to a tempalte?

### DISCUSSION THREAD

**Discussion by the1gofer** [chhoumann/quickadd#630] [Q&A]
Is there any way to pass capture info from quickadd to a tempalte?

**Comment by chhoumann** [comment]
Coming back to this one very late - sorry about that. Short answer: yes.

Put both choices inside a Macro (the Capture step first, then the Template step). All steps in one macro run share the same variables, so a named value is only prompted once:

- In the Capture format, write `{{VALUE:myInfo}}` - QuickAdd prompts for it.
- In the Template (content, file name format, or folder path), also write `{{VALUE:myInfo}}` - it reuses the same answer without prompting again.

The named form `{{VALUE:name}}` is what reliably carries across macro steps; give the variable the same name in both places and you're set. More on this in the Macro docs: https://quickadd.obsidian.guide/docs/Choices/MacroChoice
